### ISSUE TITLE
🐛 Bug Report: An array cannot be empty anymore

### ISSUE THREAD

**Issue by dereinefinn** [appwrite/appwrite#4747]
### 👟 Reproduction steps

Create a collection and a document within
1. Create an array attribute (e.g. a String array) in a collection
2. Add a value to the array
3. Update the document
4. Remove the added value from the array
5. Update the document
6. Now the length of the array is 1 and the value with index 0 is just "" (in case of a String array)

### 👍 Expected behavior

If there aren't any values in the array the length should be 0.

### 👎 Actual Behavior

The array is empty but the length is 1. The value with index 0 is "". 

![image](https://user-images.githubusercontent.com/42511728/203172148-f56c01cf-a535-4cbc-8356-786825f2e499.png)
![image](https://user-images.githubusercontent.com/42511728/203172252-e5935924-76be-4aa5-8429-3c621f7b958d.png)
![image](https://user-images.githubusercontent.com/42511728/203172305-bd0dd214-46c4-45ed-b227-272410992ba1.png)

### 🎲 Appwrite version

Version 1.1.x

### 💻 Operating system

Linux

### 🧱 Your Environment

Flutter Appwrite SDK version 8.1.0

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@dereinefinn, thanks for raising this issue! 🙏🏼 It seems this only pertains to our new Console and it's partly due to the difficulty in distinguishing whether there should be 0 elements or an element with an empty string, is that correct?

**Comment by dereinefinn** [comment]
Yes, the problem is that if the array should be empty there is one empty String at index 0. With console 1.0 we didn't have this issue.

**Comment by stnguyen90** [comment]
In the previous Console, adding blank values broke the UI so looks like we traded one problem for another 😅 I'll let the team know about this one

**Comment by stnguyen90** [comment]
We currently show an empty input for arrays like:

<img width="815" alt="image" src="https://user-images.githubusercontent.com/1477010/203183503-b9f70ac8-47b3-4aaf-8b08-2e79079e916c.png">

We should probably do something like Permissions:

<img width="1206" alt="image" src="https://user-images.githubusercontent.com/1477010/203183611-5a582050-8eac-42cd-ade2-8b95f357335e.png">

where we don't show any input for an empty array.

**Comment by ddenev** [comment]
Just to add more info to the issue:

When creating a NEW document via the Console (2.0) and not providing any value for the string array attribute, the string array attribute is created as `[null]` (i.e. an array with one element that is `null`) in the database.

Appwrite version 1.2

**Comment by JakeAi** [comment]
@stnguyen90 can confirm this issue also broke my website and the only way to fix it is not using the console...
